Enable image caching

stp-ip opened the pull request:
Remove usage of always pulling the images.
No need to request the image every time
no authentication needed and image is versioned.

luna-duclos commented:
This is totally reasonable, can you remove the `ImagePolicy: Always` tag for the other container as well ?

stp-ip committed:
Enable image caching

Remove usage of always pulling the image.
No need to request the image every time
no authentication needed and image is versioned.

stp-ip commented:
Missed that one. Pushed a new commit.
